### ISSUE TITLE
Block indentation not handled correctly

### ISSUE THREAD

**Issue by g1ps** [adrienverge/yamllint#170]
**Environment**
Cygwin on Win 7 Pro
```
$ yamllint -v
yamllint 1.15.0
```

**zz.yml**
```
---
- hosts: xyz

  tasks:
  - name: block 1
    block:
    - name: set facts 1
      set_fact:
        fact_1: 'something'
      when: var1 is undefined or var2 == ''

    - name: set facts 2
      set_fact:
        fac2: '{{ j1 }}'
      when: var1 is defined and var1 != ''
```

.yamlint
```

extends: default

rules:
  empty-lines:
    max: 0
    max-end: 1
    max-start: 0
  line-length:
    max: 250
  new-lines: disable  # Newlines might be either depending on the OS perforce checks out to.
  truthy: disable     # Ansible seems to have standardised on yes/no.
```

**Output**
$ yamllint ./zz.yml
./zz.yml
  6:12      error    syntax error: mapping values are not allowed here

**Expected Output**
yamllint should flag the block as needing to be indented, not outdented. Moving the block's contents either way removes the warning but the report is wrong. I also tried:
```
  indentation:
    indent-sequences: true
```
but that doesn't change the results. The docs are unclear as to the default state of params like this. They say only that the default is `enabled` but not what that means for the parameters to `indentation`.

As a side note, `yamllint` also doesn't pick up indentation in `tasks` when it's used like this. It seems only to detect it when I use the hyphen before `tasks` or in some inconsistent way which I can't quite pin down.

**Comment by g1ps** [comment]
I just noticed I've actually shown teh wrong error here. That's not the one I was meaning to show.

**Comment by adrienverge** [comment]
I can't reproduce. Please reopen the issue if needed.
